### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by himanshu007-creator**
#681

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@himanshu007-creator

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: documenting.rst [hunk 1]
```diff
@@ -1679,8 +1679,8 @@ in production, other are work in progress:
 .. _mail_it: https://mail.python.org/pipermail/doc-sig/2019-April/004114.html
 .. _mail_lt: https://mail.python.org/pipermail/doc-sig/2019-July/004138.html
 .. _mail_ru: https://mail.python.org/pipermail/doc-sig/2019-May/004131.html
-.. _tx_pl: https://www.transifex.com/python-doc/python-newest/language/pl/
-.. _tx_zh_cn: https://www.transifex.com/python-doc/python-newest/language/
+.. _tx_pl: https://www.transifex.com/python-doc/python-newest/
+.. _tx_zh_cn: https://www.transifex.com/python-doc/python-newest/
```
Comment (Mariatta) [inline documenting.rst:1683]: Does this look correct, @JulienPalard? Are there better urls for these?

**Review by Mariatta — APPROVED** [review]
Looks great overall, thanks. Just wasn't sure about the transifex urls.

**Comment by himanshu007-creator** [comment]
@Mariatta , mam i am a big fan of yours!😬. i read the GitHub readme project and there your profile was featured showing that people contributing for mars  copter will get a badge, i stalked your GitHub and realized, you have done great work!.
Really felt awesome to work with you, 🌸

File: documenting.rst (hunk 1 above)
Comment (JulienPalard) [reply documenting.rst:1683]: This is complicated. Transifex give 404 when someone is not logged (I don't like it).

So https://www.transifex.com/python-doc/python-newest/language/pl/ and https://www.transifex.com/python-doc/python-newest/language/zh_CN/ are the real URLs, which only work for logged in users. Maybe  https://www.transifex.com/python-doc/python-newest/ is more friendly, as I don't expect much people to already be logged in on transifex when clicking the link.

**Review by JulienPalard — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks for the kind message and for your contribution!

**Pull request merged**